lib/infoblox/resource/zone.rb

```diff
@@ -0,0 +1,8 @@
+module Infoblox 
+  class Zone < Resource
+    attr_accessor :fqdn 
+    remote_attr_accessor :address, :comment, :disable, :network_view, :extattrs
+    
+    wapi_object "zone"
```

Comment:
I don't see the Zone object in the documentation for WAPI 2.0 - is it undocumented?  I see several objects whose names start with "Zone" but not "Zone" itself.